### DISCUSSION TITLE
Added: LDM Notes for May 13, 2019

### DISCUSSION THREAD

**Discussion by jcouv** [dotnet/csharplang#2527] [General]
https://github.com/dotnet/csharplang/blob/master/meetings/2019/LDM-2019-05-13.md

Agenda:
Close on desired rules for warning suppressions and #nullable interacting

**Comment by Joe4evr** [comment]
Oh nice, so `#nullable enable annotations` is a solution to the declaration-site of the DTO problem.

Now I hope that the next update to Nullable will add something that makes the compiler warn at construction-sites (#2328).
